rotkehlchen/api/v1/resources.py

```diff
@@ -2606,3 +2606,9 @@ class AllNamesResource(BaseMethodView):
     @use_kwargs(post_schema, location='json')
     def post(self, addresses: List[ChecksumEthAddress]) -> Response:
         return self.rest_api.search_for_names_everywhere(addresses=addresses)
+
+
+class ConfigurationResource(BaseMethodView):
```

Comment:
```suggestion
class ConfigurationsResource(BaseMethodView):
```
Looks better. What do you think?